data/landSales/getLandSales.ts

```diff
@@ -272,3 +274,44 @@ export async function getLandSaleFiles(presale: string, networkName: string): Pr
   const prices = (await import(`./${presale}/prices`)).default;
   return {secret, sectors, bundles, prices}
 }
+
+export function getDeadline(hre: HardhatRuntimeEnvironment, sector: number): number {
+  let deadline = deadlines[sector];
+  if (!deadline) {
+    throw new Error(`no deadline for sector ${sector}`);
+  }
+  if (hre.network.tags.testnet) {
+    hre.deployments.log('increasing deadline by 1 year');
+    deadline += 365 * 24 * 60 * 60; //add 1 year on testnets
+  }
+  return deadline;
+}
+
+export function writeProofs(hre: HardhatRuntimeEnvironment, landSaleName: string, landSale: LandSale): void {
+  if (hre.network.name !== 'hardhat' || landSaleName === "EstateSaleWithAuth_0_0") {
```

Comment:
I'm confused by this condition 
```
landSaleName === "EstateSaleWithAuth_0_0"
```
Why do we need it ? And why not in the deploy script ?

Reply:
I'm using the proofs to call the contract, but the proofs are not written to disk during tests...

Proofs are generated at deployment time, not sure how else to get them

Reply:
I meant to move out the condition in the deploy script
something like that ?
```    
if (hre.network.name !== 'hardhat' || sector === "0") {
      writeProofs(hre, landSaleName, landSale);
 }
 ```

Reply:
Sure, that will work too, I extracted into this function because it was repetead 6 times in previous deploy scripts.

I hope we don't need to write any new deploy scripts with the new one, so I can move it over to the deploy func if really needed.